apps/sens/fsw/unit_test/sens_test_utils.cpp

```diff
@@ -73,9 +240,79 @@ void SENS_Test_Setup(void)
     Ut_OSAPI_Reset();
     Ut_OSFILEAPI_Reset();
 
-    Ut_CFE_TBL_AddTable(SENS_CONFIG_TABLE_FILENAME, (void *) &ConfigTbl);
+    if (0)
```

Comment:
What is the purpose of this pattern?

Reply:
I will remove unused macros. This was for to checking both external config table and config table for unit test.
I will use the external config table which is used by the SENS application and use the config table for unit test only for specific test cases which need it.

Reply:
Done